plugins/org.jkiss.dbeaver.model.lsm/grammar/SQLStandardLexer.g4

```diff
@@ -328,6 +328,7 @@ ZONE: Z O N E ;
 
 
 // symbols
+TypeCast: '::';
```

Comment:
It's not a part of the SQL standard. Is it OK that it's included here?

Reply:
I see, okay, I thought the word "Standard" in the grammar was supposed to denote this grammar as SQL standard-compliant.